test/LibraryManager.IntegrationTest/Services/InstallDialogVerifier.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.Test.Apex.VisualStudio;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest.Services
+{
+    public class InstallDialogVerifier : VisualStudioInProcessTestExtensionVerifier
```

Comment:
>InstallDialogVerifier [](start = 17, length = 21)

What does this class do?

Reply:
This doesn't do anything but the VisualStudioInProcessTestExtension needs a verifier. I couldn't find anything that I could reuse. Let me know if you have any suggestions.

---
In reply to: [205518541](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205518541) [](ancestors = 205518541)

Reply:
I assume you could just use VisualStudioInProcessTestExtensionVerifier directly instead of making this class?

---
In reply to: [205577425](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205577425) [](ancestors = 205577425,205518541)

Reply:
VisualStudioInProcessTestExtensionVerifier is abstract so it doesn't work

---
In reply to: [205598218](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205598218) [](ancestors = 205598218,205577425,205518541)

Reply:
Gotcha, kinda sucky though

---
In reply to: [205616154](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205616154) [](ancestors = 205616154,205598218,205577425,205518541)